llvm/tools/llvm-cxxfilt/llvm-cxxfilt.cpp

```diff
@@ -64,7 +66,15 @@ static void error(const Twine &Message) {
   exit(1);
 }
 
-static std::string demangle(const std::string &Mangled) {
+// Quote Undecorated with "" if asked for and not already followed by a '"'
+static std::string optionalQuote(std::string Undecorated,
```

Comment:
Should `Undecorated` be a `const std::string &` or `StringRef`?

Reply:
I think it's likely this function call will get optimized away, so it's somewhat moot, and we should just do what requires less mental logistics by readers of the code (which typically would not expect a string to be passed by value).